packages/vaadin-crud/test/crud.test.js

```diff
@@ -864,6 +864,25 @@ describe('crud', () => {
       expect(crud.$.toolbar.style.display).to.be.not.equal('none');
     });
 
+    it('should have hide-toolbar attribute when property set to true', () => {
```

Comment:
We should probably create a visual test instead to make sure that the toolbar is hidden with `hide-toolbar` prop set. (as reflecting to an attribute is already covered with `reflectToAttribute`)